website/source/docs/drivers/docker.html.md

```diff
@@ -105,6 +105,8 @@ The following options are available for use in the job specification.
 
 * `interactive` - (Optional) `true` or `false` (default). Keep STDIN open on
   the container.
+  
+* `shm_size` - (Optional) The size (bytes) of /dev/shm for the container.
```

Comment:
@diptanu Is there any additional documentation that should be updated or is this sufficient?